crates/tx5/src/lib.rs

```diff
@@ -9,20 +9,17 @@
 //!
 //! Tx5 can be backed currently by 1 of 2 backend webrtc libraries.
 //!
-//! - <b><i>`*`DEFAULT`*`</i></b> `backend-go-pion` - The pion webrtc library
+//! - <b><i>`*`DEFAULT`*`</i></b> `backend-libdatachannel` - WebRTC library
+//!   writen in c++.
```

Comment:
Not sure which is generated and which is original here :)

```suggestion
//!   written in C++.
```

Reply:
Done: https://github.com/holochain/tx5/pull/147/commits/8d7268bcf846c8721e74c172bf245ca9da3c094d